### PR TITLE
Add Romanian strings

### PR DESCRIPTION
**Pull request by tmihai20**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: resources/language/resource.language.ro_ro/strings.po [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+msgid "Another device handles the auto-updates of shared-library"
+msgstr "Alt dispozitiv gestionează actualizările automate ale librăriei partajate"
+
+msgctxt "#30212"
+msgid "There is no device that handles the auto-updates of shared-library"
+msgstr "Nu există dispozitiv care să gestioneze actualizările automate ale librăriei partajate"
+
+msgctxt "#30213"
+msgid "InputStream Helper settings..."
+msgstr "Setări add-on InputStream Helper"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:843]: add ending dots "..." for coherence like you did above

File: resources/language/resource.language.ro_ro/strings.po [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+msgid "Owner account"
+msgstr "Cont principal"
+
+msgctxt "#30222"
+msgid "Kid account"
+msgstr "Cont copil"
+
+msgctxt "#30223"
+msgid "Synchronization now uses \"My List\" of profile {}"
+msgstr "Sincronizarea folosește acum 'Lista mea' aferent[ profilului {}"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:883]: The char [ is intentional?

File: resources/language/resource.language.ro_ro/strings.po [hunk 3]
```diff
@@ -0,0 +1,947 @@
...
+msgid "Content for {} will require the PIN to start playback."
+msgstr "Conținutul pentru {} va avea nevoie de cod pentru a porni redarea."
+
+msgctxt "#30234"
+msgid "Install Up Next add-on"
+msgstr "Instalaează add-on-ul Up Next"
+
+msgctxt "#30235"
+msgid "[WIP] Synchronize the watched status of the videos with Netflix"
+msgstr "[Muncă în desfășurare] Sincronizează statutul conținutului urmărit cu Netflix"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:931]: isn't there an abbreviation like WIP?

File: resources/language/resource.language.ro_ro/strings.po [hunk 4]
```diff
@@ -0,0 +1,947 @@
...
+msgid "View for episodes"
+msgstr "Mod vizualizare pentru episoade"
+
+msgctxt "#30044"
+msgid "View for profiles"
+msgstr "Mod vizualizare pentru profile"
+
+msgctxt "#30045"
+msgid "Thumb rating removed|You rated a thumb down|You rated a thumb up"
+msgstr "Evaluare ștearsă|Evaluare negativ[|Evaluare pozitivă"
```
[the remark this answers is not in the document]
Comment (tmihai20) [reply resources/language/resource.language.ro_ro/strings.po:199]: No, it is a mistake, I will correct it.

File: resources/language/resource.language.ro_ro/strings.po [hunk 5]
```diff
@@ -0,0 +1,947 @@
...
+
+msgctxt "#30004"
+msgid "Password"
+msgstr "Parolă"
+
+msgctxt "#30005"
+msgid "E-mail"
+msgstr "Adresă email"
+
+# Unused 30006
```
[the remark this answers is not in the document]
Comment (tmihai20) [reply resources/language/resource.language.ro_ro/strings.po:51]: What is missing here?

File: resources/language/resource.language.ro_ro/strings.po (hunk 3 above)
Comment (tmihai20) [reply resources/language/resource.language.ro_ro/strings.po:931]: There is not any in Romanian. I will shorten it.

File: resources/language/resource.language.ro_ro/strings.po (hunk 5 above)
[the remark this answers is not in the document]
Comment (tmihai20) [reply resources/language/resource.language.ro_ro/strings.po:51]: I see now, string 30006 is missing, I will add it.

**Comment by tmihai20** [comment]
I made a new commit fixing what you noticed and I also updated some translated strings.

File: resources/language/resource.language.ro_ro/strings.po [hunk 6]
```diff
@@ -0,0 +1,954 @@
...
+msgid "Netflix returned an unknown error."
+msgstr "Netflix a întâlnit o eroare necunoscută."
+
+msgctxt "#30102"
+msgid "Netflix returned the following error:"
+msgstr "Netflix a dat de următoarea eroare:"
+
+msgctxt "#30103"
+msgid "Error details have been written to the Kodi log file. If the error message does not provide guidance on how to resolve the issue or you continue to receive this error, please report it on GitHub by following the Readme instructions."
+msgstr ""
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:409]: empty msgstr to remove

File: resources/language/resource.language.ro_ro/strings.po [hunk 7]
```diff
@@ -0,0 +1,954 @@
+# Kodi Media Center language file
+# Addon Name: Netflix
+# Addon id: plugin.video.netflix
+# Addon Provider: libdev + jojo + asciidisco + caphm + CastagnaIT
+msgid ""
+msgstr ""
+"Project-Id-Version: plugin.video.netflix\n"
+"Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
+"POT-Creation-Date: 2017-07-24 16:15+0000\n"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:9]: update also creation date thanks

File: resources/language/resource.language.ro_ro/strings.po [hunk 8]
```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: plugin.video.netflix\n"
 "Report-Msgid-Bugs-To: https://github.com/CastagnaIT/plugin.video.netflix\n"
-"POT-Creation-Date: 2017-07-24 16:15+0000\n"
-"PO-Revision-Date: 2020-04-13 19:54+0000\n"
+"POT-Creation-Date: 2017-07-13 19:54+0000\n"
```
Comment (CastagnaIT) [inline resources/language/resource.language.ro_ro/strings.po:9]: 2017 return to the future?

**Pull request merged**